recipe/bld.bat

```diff
@@ -12,3 +12,38 @@ if errorlevel 1 exit 1
 
 copy %SRC_DIR%\lib\%SLN_PLAT%\blitz.dll %LIBRARY_BIN%\
 copy %SRC_DIR%\lib\%SLN_PLAT%\blitz.lib %LIBRARY_LIB%\
+
+REM Copy headers
+mkdir %LIBRARY_INC%\blitz
+mkdir %LIBRARY_INC%\blitz\array
+mkdir %LIBRARY_INC%\blitz\gnu
+mkdir %LIBRARY_INC%\blitz\meta
+mkdir %LIBRARY_INC%\blitz\ms
+
+copy %SRC_DIR%\blitz\*.h %LIBRARY_INC%\blitz\
+copy %SRC_DIR%\blitz\*.cc %LIBRARY_INC%\blitz\
+copy %SRC_DIR%\blitz\array\*.h %LIBRARY_INC%\blitz\array\
+copy %SRC_DIR%\blitz\array\*.cc %LIBRARY_INC%\blitz\array\
+copy %SRC_DIR%\blitz\gnu\*.h %LIBRARY_INC%\blitz\gnu\
+copy %SRC_DIR%\blitz\gnu\*.cc %LIBRARY_INC%\blitz\gnu\
+copy %SRC_DIR%\blitz\meta\*.h %LIBRARY_INC%\blitz\meta\
+copy %SRC_DIR%\blitz\meta\*.cc %LIBRARY_INC%\blitz\meta\
+copy %SRC_DIR%\blitz\ms\*.h %LIBRARY_INC%\blitz\ms\
+copy %SRC_DIR%\blitz\ms\*.cc %LIBRARY_INC%\blitz\ms\
```

Comment:
Why are you copying the .cc files?
Also you need to bump the build number too to get a new package:
https://conda-forge.org/#update_recipe

Reply:
Valid point.